### ISSUE TITLE
Are types working in v0.3.0?

### ISSUE THREAD

**Issue by zackarydev** [SegFaultx64/typescript-graph#2]
Seems like types cannot be found in version 0.3.0.

Perhaps due to package.json -> `types` property?
`  "typings": "dist/types/index.ts",`

Path `/dist/types/index.d.ts` once bundled

**Comment by skoshx** [comment]
Yeah, not working... Quick fix is to just change `typings` from `dist/types/index.ts` -> `dist/types/index.d.ts`, then save a patch with `patch-package`...

@SegFaultx64 Fix coming or should I open a PR?

**Comment by dhcheung** [comment]
Ran into this issue too, found a different workaround that doesn't need a patch

in your tsconfig.json, add this path override
```
{
    "compilerOptions": {
        // ... your other options
        "paths": {
          "typescript-graph": ["./node_modules/typescript-graph/dist/types/index.d.ts"]
        }
    }
}
```
